### PR TITLE
Add component SymptomsTracker.js

### PR DESCRIPTION
**Pull request by pavel-ilin**
Add component SymptomsTracker.js

Done:
Functional part. Form ready to collect and submit user input.

Problems:
- Need arrange elements in symmetrical order
alt text
- Need to add selected button style
- Need to synchronize css with DiagnosticContainer.js component. Right now main menu blocked half of the form
alt text

Next:
Need to dispatch redux action and send collected data to the backend

### CODE REVIEW

File: client/src/components/SymptomsTracker.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+
+    const classes = useStyles();
+
+    const [todayFeeling, setTodayFeeling] = useState(1)
+    const [todaySymptoms, setTodaySymptoms] = useState(1)
+    const [comparedFeeling, setcomparedFeeling] = useState(1)
+    const [cough, setCough] = useState('')
+    const [fever, setFever] = useState(0)
+    const [dizinnes, setDizinnes] = useState('')
+    const [soarThroat, setSoarThroat] = useState('')
```
[the remark this answers is not in the document]
Comment (SomeMoosery) [reply client/src/components/SymptomsTracker.js]: It looks like there's still 4 instances of soar (lines 118, 199, 202) unless I'm looking at an old commit 🤔

[the remark this answers is not in the document]
Comment (pavel-ilin) [reply client/src/components/SymptomsTracker.js]: In rebase commit it all should be 'sore'

[the remark this answers is not in the document]
Comment (SomeMoosery) [reply client/src/components/SymptomsTracker.js]: In the `Files Changed` section it looks like the rebase didn't go as it should've (classic git... 🙄), I'm still seeing those references of soar as well as deleted context for routing (for example, in `App.js`, `BroserRouter` and related elements are being removed). 

I can take a look in a little bit, or maybe something is just wrong on my end? If you're seeing router stuff and not seeing any instances of soar I'm not sure what's up. You rebased off of the most current copy of master right?

**Comment by SomeMoosery** [comment]
Going to merge these changes in so we can start getting to work on dealing with questions - there's few minor tweaks regarding `package.json` having Babel 7.8.7 back in it which it shouldn't due to it being a broken release by Babel that I'll make later this afternoon as they won't affect us moving forward for now

**Pull request merged**